src/App.css

```diff
@@ -0,0 +1,21 @@
+/* Typography system */
+h1,
+h2 {
+  font-family:
+    "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+    "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+}
+
+/* Color system */
+
+:root {
+  --primary-color: blue;
+  --background-color: antiquewhite;
+  --text-color: purple;
+  --text-secondary-color: darkgrey;
+  --button-color: blue;
+  --input-background-color: green;
```

Comment:
It's not a common practice to used the named colors. You don't need to change this for the class but I recommend reading over this article by Josh Comeau: [Color Formats in CSS - hex, rgb, hsl, lab](https://www.joshwcomeau.com/css/color-formats/)